### PR TITLE
Add a dashboard for torchao

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #140487

Summary:
trying to add a similar dashboard as https://hud.pytorch.org/benchmark/llms?repoName=pytorch%2Fpytorch

Test Plan:
Run locally:

install torchao
```
pip install --no-use-pep517 --user "git+https://github.com/pytorch/ao.git@f96e5ec9d30f778c2fb49d7ff9536b8016e1d408"
```
run test:
```
cd benchmarks/torchao
python benchmark.py
```

Check after landing
https://hud.pytorch.org/benchmark/llms?repoName=pytorch%2Fpytorch

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

File: benchmarks/torchao/benchmark.py [hunk 1]
```diff
@@ -0,0 +1,308 @@
+import argparse
```
Comment (yanboliang) [inline benchmarks/torchao/benchmark.py:1]: Is there a strong reason that we need to duplicate this folder? Can we just refer the original one?

Comment (jerryzh168) [reply benchmarks/torchao/benchmark.py:1]: what do you mean by original one?

Comment (jerryzh168) [reply benchmarks/torchao/benchmark.py:1]: Sounds good, I can use gpt-fast

**Comment by jerryzh168** [comment]
will be using https://github.com/pytorch/pytorch/pull/140627 instead

**Pull request closed without merging**